src/pkcs11/pkcs11-spy.c

```diff
@@ -1462,7 +1501,7 @@ C_WrapKey(CK_SESSION_HANDLE hSession, CK_MECHANISM_PTR pMechanism,
 	spy_dump_ulong_in("hWrappingKey", hWrappingKey);
 	spy_dump_ulong_in("hKey", hKey);
 	rv = po->C_WrapKey(hSession, pMechanism, hWrappingKey, hKey, pWrappedKey, pulWrappedKeyLen);
-	if (rv == CKR_OK)
+	if (rv == CKR_OK || rv == CKR_BUFFER_TOO_SMALL)
```

Comment:
here is one instance that you missed

Reply:
Thanks . Changed also this one (and rebased on master to fix Fedora CI).